filters/encrypt/docs_test.go

```diff
@@ -0,0 +1,109 @@
+package encrypt_test
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/hashicorp/eventlogger"
+	"github.com/hashicorp/eventlogger/filters/encrypt"
+	"github.com/hashicorp/eventlogger/sinks/writer"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/hashicorp/go-kms-wrapping/wrappers/aead"
+)
+
+func ExampleFilter() {
+	then := time.Date(
+		2009, 11, 17, 20, 34, 58, 651387237, time.UTC)
```

Comment:
What's the significance of this date?  :-D

Reply:
It is Boundary's birthday

Reply:
On phone and saw this comment come in.  We should totally update this PR to make it Boundary’s real birthday